content/tv/code/0006.md

```diff
@@ -8,16 +8,12 @@ minutes: 120
 aliases: ["/tv/code/6"] #redirect alias
 episode: "6"
 # Run this after the show is uploaded to youtube
```

Comment:
Why delete the comments? It's easier to copy paste to new ones with it. Also, you forgot a comment.

Reply:
We don't use the youtube image for code, we keep it at the episode image. I was cleaning up to match the template ( https://raw.githubusercontent.com/vmware-tanzu/tanzu-dev-portal/main/content/tv/code/_template.md )